stable/database/values.yaml

```diff
@@ -390,6 +393,9 @@ database:
           accessModes:
             - ReadWriteOnce
           # storageClass: "-"
+          dataSourceRef:
```

Comment:
Can you a comment with a link to the Kubernetes documentation so that we know what the schema is what this is used for, e.g.
```
# The data source for the archive PVC
# see https://kubernetes.io/docs/concepts/storage/persistent-volumes/#data-source-references
```

We should also include these in the README. We want to be comprehensive in our descriptions in the README so that people know how to supply these values.